docs/flashbots-core/releases/v0.2-release.mdx

```diff
@@ -0,0 +1,163 @@
+
+A searcher using an execution contract for their trades which hardcodes block.coinbase as the payment recipient will need to deploy a new contract which can send the payment to a configurable address with a configurable gas budget.
+
+The address and gas consumption of the router contract is likely to change in the future but the ABI is expected to remain backwards compatible.
+
+#### Bundle scoring
+Bundles received by mev-geth will use a new scoring function to [mitigate some of the issues](https://hackmd.io/@flashbots/core-v2-proposal#Revamped-auction-pricing) with the scoring function used in v0.1.
+
+While the v0.2 pre-release removed the ability to pay the miner using gas price of a transaction, the scoring function selected for v0.2 allows the use of gas price based payments but excludes payments from transactions which are present in the regular transaction pool.
+
+Here is the formal definition of these scoring functions:
+
+<Tabs
+  defaultValue="v0.1"
+  values={[
+    { label: 'v0.1', value: 'v0.1', },
+    { label: 'v0.2-pre', value: 'v0.2-pre' },
+    { label: 'v0.2', value: 'v0.2' }
+  ]}
+>
+<TabItem value="v0.1">
+
+$$
+s_{v0.1} = \frac{\Delta_{coinbase} + \sum_{T\in U}g_Tp_T}{\sum_{T\in U}g_T}
+$$
+</TabItem>
+<TabItem value="v0.2-pre">
+
+$$
+s_{v0.2-pre} = \frac{\Delta_{coinbase}}{\sum_{T\in U}g_T}
+$$
+</TabItem>
+<TabItem value="v0.2">
+
+$$
+s_{v0.2} = \frac{\Delta_{coinbase} + \sum_{T\in U}g_Tp_T - \sum_{T\in M \cap U}g_Tp_T}{\sum_{T\in U}g_T}
+$$
+</TabItem>
+</Tabs>
+
+$s$: bundle $U$ _score_ used to sort bundles.  
+$U$: ordered list of transactions $T$ in a bundle.  
+$M$: set of transactions $T$ in the mempool.  
+$g_{T}$: _gas used_ by transaction $T$.  
+$p_{T}$: _gas price_ of transaction $T$.  
+$\Delta_{coinbase}$: coinbase difference from direct payment.  
+  
+#### Bundle validation
+The relay removes the temporary limitation of two 'from' addresses per bundle. The bundle can once again contain transactions from an arbitrary number of EOAs.
+
+#### Bundle delivery
+The `eth_sendBundle` RPC method on mev-geth is put into maintenance mode in the v0.2 release. Instead, mev-geth establishes an outgoing websockets connection with the relay and streams bundles as they become available. This improves the security of the miner infrastructure as it no longer requires exposing an incoming RPC port.
+
+To connect to the relay over websockets, each miner will receive an access key which they must add to their mev-geth node to perform the authentication.
+
+#### Block construction
+A new parameter `revertingTxHashes` is added to the bundles which allows the searcher to list the transactions in their bundle which are allowed to revert. By default, all transactions included in a bundle should never revert on chain.
```

Comment:
Shouldn't this go in the above "`eth_sendBundle` RPC parameters change" section?